### ISSUE TITLE
关于parser参数中使用了dropout，但是在代码的其它部分却没有使用到这个参数

### ISSUE THREAD

**Issue by 1456416403** [ShannonAI/ChineseBert#57]
parser.add_argument("--hidden_dropout_prob", default=0.1, type=float, help="dropout probability")
代码中设置了 hidden_dropout_prob，但是却通篇找不到使用了参数的地方

比如
        self.bert = GlyceBertModel(config)
        self.dropout = nn.Dropout(config.hidden_dropout_prob)
        self.classifier = nn.Linear(config.hidden_size, config.num_labels)
modeling_glycebert.py中253行，直接使用了hidden_dropout_prob
